sql/mysqld.cc

```diff
@@ -9200,6 +9218,7 @@ void refresh_status(THD *thd)
     connections.  This is not perfect, but status data is not exact anyway.
   */
   max_used_connections= connection_count + extra_connection_count;
+  max_used_connections_time= time(nullptr);
```

Comment:
could use `thd->{timevar}` here instead of `time` too.

Reply:
@grooverdan, which of the (many :cold_sweat:) timestamps in `class THD` would be relevant here?

https://github.com/mariadb/server/blob/11.0/sql/sql_class.h#L2854-L2859

Reply:
> I was looking a lower overhead

Consulting the `SHOW STATUS` output is not in any plausible circumstance a fast-path operation… is it?